sklearn/utils/estimator_checks.py

```diff
@@ -411,13 +411,14 @@ def check_fit2d_1sample(name, Estimator):
     set_random_state(estimator, 1)
     try:
         estimator.fit(X, y)
-    except ValueError:
-        pass
-
+    except:
+        # The message should say something about only one sample
+        msg = "sample"
+        assert_raise_message(ValueError, msg, estimator, X, y)
```

Comment:
that is not right, sorry I was not having a proper look. We already ran ``fit`` and caught the exception. You need to check the exception similar to this: https://github.com/sotlampr/scikit-learn/blob/master/sklearn/utils/estimator_checks.py#L326